nginx-config/nijobs.conf

```diff
@@ -0,0 +1,37 @@
+server {
+        # listen on port 80
+        listen 80;
+        # save logs here
+        access_log /var/log/nginx/access.log main;
+
+        # where the root here
+        root /usr/share/nginx/html;
```

Comment:
This can be excluded, as it is the default, I believe

Reply:
From what I've seen, a lot of examples had /var/www as default, we should keep it in order to make it clearer

Reply:
The fact that the Dockerfile worked without an nginx.conf disproves that. The default for nginx's static file serving is `/usr/share/nginx/html`

Reply:
That might be true, however, it doesn't hurt to have it there to make it clearer for future devs.

Reply:
Sure, I'm fine with having it this way